apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -173,11 +181,17 @@ var TopInstructions = React.createClass({
       this.props.noVisualization && styles.noViz,
       this.props.isEmbedView && styles.embedView,
     ];
+    const ttsUrl = this.props.ttsMarkdownInstructionsUrl;
 
     return (
       <div style={mainStyle} className="editor-column">
         <PaneHeader hasFocus={false}>
+
           <div style={styles.paneHeaderOverride}>
+            {experiments.isEnabled('CSDTTS') &&
+            <div id="foundIt!" style={styles.audioControls}>
```

Comment:
nit: exclamation marks in ids, although technically allowed by HTML5 (but not 4), are still a little weird.